flink-formats/flink-avro/src/test/java/org/apache/flink/formats/avro/AvroSplittableInputFormatTest.java

```diff
@@ -212,38 +211,38 @@ public void createFiles() throws IOException {
     }
 
     @Test
-    public void testSplittedIF() throws IOException {
+    void testSplittedIF() throws IOException {
         Configuration parameters = new Configuration();
 
         AvroInputFormat<User> format =
                 new AvroInputFormat<>(new Path(testFile.getAbsolutePath()), User.class);
 
         format.configure(parameters);
         FileInputSplit[] splits = format.createInputSplits(4);
-        assertEquals(splits.length, 4);
+        assertThat(4).isEqualTo(splits.length);
         int elements = 0;
         int[] elementsPerSplit = new int[4];
         for (int i = 0; i < splits.length; i++) {
             format.open(splits[i]);
             while (!format.reachedEnd()) {
                 User u = format.nextRecord(null);
-                Assert.assertTrue(u.getName().toString().startsWith(TEST_NAME));
+                assertThat(u.getName().toString().startsWith(TEST_NAME)).isTrue();
                 elements++;
                 elementsPerSplit[i]++;
             }
             format.close();
         }
 
-        Assert.assertEquals(1604, elementsPerSplit[0]);
-        Assert.assertEquals(1203, elementsPerSplit[1]);
-        Assert.assertEquals(1203, elementsPerSplit[2]);
-        Assert.assertEquals(990, elementsPerSplit[3]);
-        Assert.assertEquals(NUM_RECORDS, elements);
+        assertThat(elementsPerSplit[0]).isEqualTo(1604);
+        assertThat(elementsPerSplit[1]).isEqualTo(1203);
+        assertThat(elementsPerSplit[2]).isEqualTo(1203);
+        assertThat(elementsPerSplit[3]).isEqualTo(990);
```

Comment:
```suggestion
        assertThat(elementsPerSplit).containsExactly(1604, 1203, 1203, 990);
```

Reply:
:+1: I fixed a couple of these just after creating the PR -- you reviewed very quickly! :D